coops-ui/src/components/AnnualReport/BCorp/ARDate.vue

```diff
@@ -0,0 +1,40 @@
+<template>
```

Comment:
There's a merge/rebase snafu somewhere here, because these are FW's changes, not yours.